corehq/motech/dhis2/tests/test_dbaccessors.py

```diff
@@ -0,0 +1,61 @@
+from django.test import TestCase
+
+from corehq.motech.dhis2.const import SEND_FREQUENCY_MONTHLY
+from corehq.motech.dhis2.dbaccessors import get_migrated_dataset_maps
+from corehq.motech.dhis2.models import DataSetMap, SQLDataSetMap
+from corehq.motech.models import ConnectionSettings
+
+
+class MigrationTest(TestCase):
+
+    domain = 'test-domain'
+
+    def setUp(self):
+        self.connx = ConnectionSettings.objects.create(
+            domain=self.domain,
+            name='test connection',
+            url='https://dhis2.example.com/'
+        )
+        self.dataset_map = DataSetMap.wrap({
+            'domain': self.domain,
+            'connection_settings_id': self.connx.id,
+            'ucr_id': 'c0ffee',
+            'description': 'test dataset map',
+            'frequency': SEND_FREQUENCY_MONTHLY,
+            'day_to_send': 5,
+            'org_unit_column': 'org_unit_id',
+            'datavalue_maps': [{
+                'column': 'foo_bar',
+                'data_element_id': 'foo456789ab',
+                'category_option_combo_id': 'bar456789ab',
+            }, {
+                'column': 'foo_baz',
+                'data_element_id': 'foo456789ab',
+                'category_option_combo_id': 'baz456789ab',
+            }, {
+                'column': 'foo_qux',
+                'data_element_id': 'foo456789ab',
+                'category_option_combo_id': 'qux456789ab',
+            }]
+        })
+        self.dataset_map.save()
+
+    def tearDown(self):
+        for m in SQLDataSetMap.objects.filter(domain=self.domain).all():
+            m.delete()
+        self.dataset_map.delete()
+        self.connx.delete()
```

Comment:
nit: I don't like abbreviations like this because then I have to make a mental map of `connx -> connection_setting` to remember what model this refers to.